leenlab2/reminiscence
# Add bgm and integrate mesh colliders

Added BGM music and renamed Room scene to DemoOct11. Integrated Start Screen.

# Description


### Type of change






Integration Work
## How to Integrate


# How Has This Been Tested?

- Open Main Menu scene and hit play. You should be placed in the attic room.
- Place the three objects in the right locations and ensure confirmation noise is played, particles emit from TV, and new tape is played.
- Walk into the irregularly shapes objects on the floor and ensure mesh colliders are realistic.

# Checklists
## Integration Checklist

## Review Checklist
- [] I have requested a review from at least one (1) team member (preferably someone working on a related task)
